apps/functions/applications-background-jobs/notify-subscribers/src/util.js

```diff
@@ -51,3 +52,18 @@ export function encrypt(value) {
 
 	return `${iv.toString('hex')}${encrypted.toString('hex')}`;
 }
+
+/**
+ * Encodes the links within the html for markdown
+ *
+ * @param {string} htmlString
+ * @returns {string}
+ */
+export function encodeLinksForMarkdown(htmlString) {
+	const dom = new JSDOM(htmlString);
+	const links = dom.window.document.querySelectorAll('a'); // Select all anchor elements
+	links.forEach((link) => {
+		link.href = link.href.replaceAll("'", '%27').replaceAll('(', '%28').replaceAll(')', '%29');
+	});
```

Comment:
I know it leads to a longer line but I think it would be better to run the `forEach` directly on the `dom` object. It took me a moment to figure out that `dom` is mutated by the `forEach` because `links` is an array of references back to that object. Just a bit of indirection there.

```suggestion
         dom.window.document.querySelectorAll('a').forEach(link => {
         	link.href = link.href.replaceAll("'", '%27').replaceAll('(', '%28').replaceAll(')', '%29');
         });
```